com.nordpass.NordPass.yaml

```diff
@@ -0,0 +1,66 @@
+app-id: com.nordpass.NordPass
+base: org.electronjs.Electron2.BaseApp
+base-version: '19.08'
+runtime: org.freedesktop.Platform
+runtime-version: '19.08'
+sdk: org.freedesktop.Sdk
+command: nordpass
+tags:
+  - proprietary
+finish-args:
+  - --socket=x11
+  - --device=dri
+  - --share=ipc
+  - --socket=pulseaudio
+  - --share=network
+  - --talk-name=org.freedesktop.secrets
+  - --talk-name=org.freedesktop.Notifications
+  - --talk-name=org.kde.StatusNotifierWatcher
+modules:
+  - name: squashfs
```

Comment:
Use https://github.com/refi64/unappimage instead.

Reply:
I don't think unappimage works with Snap packages. Or is there a switch for it?

Reply:
It depends. it exists because https://github.com/flatpak/flatpak/commit/cd2142888fc4c199723a0dfca1f15ea8788a5483 "breaks" system installations of extra-data apps using squashfs images. You'd need to check if it works without `--user`.

Reply:
I've tested installing the app without `--user` and it seem to both install and run fine. Is there something special to look out for?